cmd/mount.go

```diff
@@ -293,13 +293,17 @@ var mountCmd = &cobra.Command{
 
 		config.Set("mount-path", options.MountPath)
 
-		log.Crit("Starting Blobfuse2 Mount : %s on (%s)", common.Blobfuse2Version, common.GetCurrentDistro())
-		log.Crit("Logging level set to : %s", logLevel.String())
-		pipeline, err := internal.NewPipeline(options.Components)
-		if err != nil {
-			log.Err("Mount: error initializing new pipeline [%v]", err)
-			fmt.Println("failed to mount :", err)
-			Destroy(1)
+		var pipeline *internal.Pipeline
+
+		if !daemon.WasReborn() {
+			log.Crit("Starting Blobfuse2 Mount : %s on (%s)", common.Blobfuse2Version, common.GetCurrentDistro())
+			log.Crit("Logging level set to : %s", logLevel.String())
+			pipeline, err = internal.NewPipeline(options.Components)
+			if err != nil {
```

Comment:
newPipeline we need to do in background as well. Only the auth part we wanted to shut down. How does this work ?

Reply:
Why? Doesnt the old pipeline variable get inherited?

Reply:
nvm youre right. Let me look into this again